### PR TITLE
Fix Ui of CKeditor for admin also fix ul element

### PR DESCRIPTION
**Pull request by alkeshrails**
Ck Editor Ui fix and fix for `ul` `ol` list
Screenshot from 2020-04-22 15-49-38

### CODE REVIEW

File: app/admin/job_posting.rb [hunk 1]
```diff
@@ -46,7 +46,8 @@ def permitted_params
       f.input :location
       f.input :country
       f.input :remote
-      f.input :description, as: :ckeditor
+      f.label 'Description', class: 'ckeditor_label job_posting'
```
Comment (skbly7) [inline app/admin/job_posting.rb:49]: Why  it should be different class names everywhere? Instead of just one say `admin-panel` etc.

Comment (alkeshrails) [reply app/admin/job_posting.rb:49]: Actually spacing is diff on pages, so I need to set margin accordingly.

File: app/assets/stylesheets/ckeditor.scss [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+  width: 75% !important;
   overflow: hidden;
 }
 
 .cke_1.cke.cke_reset.cke_chrome.cke_ltr.cke_browser_webkit {
   margin-left: 25%;
+  margin-bottom: 2%;
+}
+
+label.ckeditor_label.job_posting, label.ckeditor_label.participation_terms {
```
Comment (piotrekpasciak) [inline app/assets/stylesheets/ckeditor.scss:11]: Are you sure you want to modify all CKEditor in AICrowd?

If you want to modify css for Admin panel CKEditor use following stylesheet:
https://github.com/AIcrowd/AIcrowd/blob/master/app/assets/stylesheets/additional_active_admin.css.scss

Comment (piotrekpasciak) [reply app/assets/stylesheets/ckeditor.scss:11]: @alkeshrails said this ckeditor.scss is used in Admin panel and front-end is usinng different css file.

**Pull request merged**